jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -97,6 +98,11 @@ You can add or delete pipelines for your project. For details see the xref:pipel
 
 - You can enable dynamic configuration using setup workflows in CircleCI. To learn about dynamic configuration, read the xref:dynamic-config#[Dynamic configuration] guide.
 
+[#only-build-pull-requests]
+=== Only build pull requests
+
+In your trigger settings, select the `PR opened or pushed to, default branch pushes, tag pushes` xref:github-app-trigger-options/#supported-trigger-options[option under the "Run on" menu].
```

Comment:
```suggestion
In your trigger settings, select the "PR opened or pushed to, default branch pushes, tag pushes" option under the "Run on" menu. For a full list of GitHub App trigger options, see the xref:github-app-trigger-options/#supported-trigger-options[GitHub App trigger options] page.

image::triggers/run-on-open.png[Run on trigger menu options]
```

Reply:
We don't always follow this rule but we should generally not just link to other sections from text, the link text generally should tell you exactly what you will find when you click the link. Does that make sense?

Reply:
https://circleci.com/docs/style/links/#cross-references